tests/assets/suites/default/index.yaml

```diff
@@ -5,8 +5,3 @@ patterns:
 cases:
     -   name: default/ltplite
         tasks: suites/default/ltplite.xml
-maintainers:
```

Comment:
Is this a part of the patch?

Reply:
Kind-of. I'm stopping accepting unknown nodes with this patch and removing nodes I forgot to remove previously, which would prevent validation now. Should've added that to the commit description.